cygnus-ngsi/conf/README.md

```diff
@@ -9,4 +9,17 @@ $ cp agent_ngsi.conf.template [APACHE_FLUME_HOME]/conf/
 
 ```
 $ cp grouping_rules.conf.template [APACHE_FLUME_HOME]/conf/
-``
+```
+
+Postgres and Postis sinks are capable of inserting native Postgres types into DB according with the following table.
```

Comment:
I think we have specific .md files per-sink. In that case, I think this piece of documentation should be better placed in that ones. What do you think?

Reply:
Moreover original issue mentions:

> In row mode, it is assumed that column types are "text" and no specific processing is needed. Every value is casted to text.

If a the end it has been implemented that way, I think that sentence is a valuable piece of documentation to be included in the proper place.

Reply:
Fixed in [106b7ab](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/106b7abb5c6d2c0c25defaccaa2dd439bfb1fa29)